packages/sdks-tests/src/tests/basic-styles.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { expect } from '@playwright/test';
+import { test, currentDirname, checkIsRN } from './helpers/index.js';
+import fs from 'fs';
+import path from 'path';
+
+test.describe('Basic styles', () => {
+  test('styles should be applied consistently across all SDKs (text and buttons)', async ({
+    page,
+    packageName,
+    sdk,
+  }) => {
+    test.fail(checkIsRN(sdk));
+    test.skip(packageName === 'react-native');
```

Comment:
you're both skipping and failing react-native 😄. I'd pick just one

Reply:
thanks!!